modules/persistent-storage-csi-vsphere-disable-storage-consequences.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-disable-storage-consequences_{context}"]
+= Consequences of disabling and enabling storage on vSphere
+
+The consequences of disabling and enabling storage on vSphere are described in the following table.
```

Comment:
Didn't we mention something about installing a third party vsphere CSI driver after disabling ours could cause issues?
@jsafrane @gnufied @mpatlasov

Reply:
Yes, we talk about it here https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/storage/using-container-storage-interface-csi#persistent-storage-csi-vsphere-install-issues_persistent-storage-csi-vsphere. 

@gcharot Do we want a reference to this?

Reply:
I don't want to mention anything of that sort. We are mostly providing this as a way to disable storage for customers, who want to may be run ODF or something else. Not necessarily run third party vSphere driver right?
Because - then we will have bigger support issues.

Reply:
I agree, let's not mention third party vsphere CSI driver here. We don't have any solid data to say if it is safe or dangerous anyway.

Reply:
the case may happen where a customer wants to remove "our" driver and use the vmware one instead. I am not against not talking about it, just stating it's a possibility

Reply:
The problem with that is - topology constraints of cloud provider also match.

Reply:
Seems like we've decided not to add this info?